src/index.js

```diff
@@ -120,13 +147,20 @@ const attack = () => {
     return `${player.name} has taken too much damage and cannot carry on! ${player.name} retreats from the battle.`;
   } else {
     switch (true) {
-    case enemy.hp <= 0:
-      return genericMsg + ' ' + enemy.msgs.defeat;
-    case enemy.hp <= fightInfo.startingEnemyHp * 0.5 && enemy.msgs.sub50[1] === false:
+    case enemy.hp <= 0: {
+      fightState = false;
+      const reward = chooseWeighted(enemy.drops);
+      addItem(reward);
+      console.log(inventory);
+      return (genericMsg + ' ' + enemy.msgs.defeat + '\n\n' + `You collect one <${reward.mat}> and place it in your inventory.`);
```

Comment:
heh you funny, i never seen a hybrid template non template string before
```suggestion
      return `${genericMsg} ${enemy.msgs.defeat}\n\nYou collect one <${reward.mat}> and place it in your inventory.`;
```